stdlib/http/src/main/ballerina/src/http/client_endpoint.bal

```diff
@@ -345,7 +358,25 @@ public type OutboundAuthConfig record {|
     OutboundAuthHandler authHandler;
 |};
 
-function initialize(string serviceUrl, ClientConfiguration config) returns HttpClient|error {
+# Client configuration for cookies.
+#
+# + enabled - User agents provide users with a mechanism for disabling or enabling cookies
+# + maxPerCookieSize - At least 4096 bytes per cookie (as measured by the sum of the length of the cookie’s name, value, and  attributes)
+# + maxCookiesPerDomain - At least 50 cookies per domain
+# + maxCookieCount - At least 3000 cookies total
```

Comment:
The same problem as above is there in the above two w.r.t. field name and description

Reply:
+1 for the changes suggested by @pubudu91 .